--- cog-0.7.2/cog.desktop.orig	2004-08-13 21:10:35.000000000 +0200
+++ cog-0.7.2/cog.desktop	2004-08-28 00:40:07.234557680 +0200
@@ -1,12 +1,16 @@
 [Desktop Entry]
 Name=GNOME Configurator
+Name[pl]=Konfigurator dla Gnome
 Comment=Edit advanced GNOME settings
 Comment[fr] = Configuration facile des options avancées de Gnome
+Comment[pl]=Edycja zaawansowanych ustawieÅ. GNOME
 Icon=cog-icon-2-48x48.png
 Exec=cog
 Terminal=false
 Type=Application
 StartupNotify=true
 Encoding=UTF-8
-Categories=GNOME;Application;System;X-Red-Hat-Base;
+Categories=GNOME;Application;System;
 X-Desktop-File-Install-Version=0.4
+
+# vi: encoding=utf-8
